client/board.go

```diff
@@ -65,6 +69,57 @@ type Board struct {
 	SLOs []string `json:"slos"`
 }
 
+// BoardPanel represents a single panel on a board.
+type BoardPanel struct {
```

Comment:
a general "are you sure?": do you mean `omitempty` in all these places here?

Reply:
yeah, all coming from PAPI too